### ISSUE TITLE
DO RouteDomain example is invalid

### ISSUE THREAD

**Issue by p-semenov-f5** [F5Networks/f5-declarative-onboarding#263]
I have DO 1.23 on BIG-IP 16.1.0

I took an example from here:
https://gitlab.f5net.com/automation-toolchain/f5-declarative-onboarding/-/blob/develop/examples/routeDomains.json

And it is not working.

More simple declaration is not working too.
```json
{
    "schemaVersion": "1.0.0",
    "class": "Device",
    "async": true,
    "Common": {
        "class": "Tenant",
        "hostname": "bigip.example.com",
        "test_RD": {
            "class": "RouteDomain",
            "id": 100,
            "bandWidthControllerPolicy": "bwcPol",
            "connectionLimit": 5432991,
            "flowEvictionPolicy": "default-eviction-policy",
            "ipIntelligencePolicy": "ip-intelligence",
            "enforcedFirewallPolicy": "enforcedPolicy",
            "stagedFirewallPolicy": "stagedPolicy",
            "securityNatPolicy": "securityPolicy",
            "servicePolicy": "servicePolicy",
            "strict": false,
            "routingProtocols": [
                "BGP"
            ]
        }
    }
}
```

**Comment by dstokesf5** [comment]
Switching this from a feature request to a bug.

**Comment by dstokesf5** [comment]
Can you provide additional information on what is not working? What was the expected behavior, and what was the actual behavior?

**Comment by p-semenov-f5** [comment]
I took that from example: https://gitlab.f5net.com/automation-toolchain/f5-declarative-onboarding/-/blob/develop/examples/routeDomains.json

As I found it refers to non existent objects

**Comment by p-semenov-f5** [comment]
I tried filter out part of them:

    "errors": [
        "transaction failed:01070734:3: Configuration error: route-domain Name /Common/theRouteDomain refers to bwc policy: bwcPol which is non-existent."
    ],

    "errors": [
        "transaction failed:0107097d:3: The specified route domain (/Common/AnotherRouteDomain) has the strict attribute set and cannot have a parent unless the parent also has the strict attribute set."
    ],

    "errors": [
        "transaction failed:01070712:3: Values (securityPolicy) specified for route domain (/Common/theRouteDomain): foreign key index (security_nat_policy_FK) do not point at an item that exists in the database."
    ],

**Comment by dstokesf5** [comment]
Those look like legitimate issues with the declaration, which means we should provide a better example. I have added AUTOTOOL-2764 to our internal product backlog to improve this example.

**Comment by dstokesf5** [comment]
This has been resolved in the 1.26.0 release.
